STMicroelectronics/STM32L4_L4+/samples/key_vault.1/key_vault_interface.c

```diff
@@ -3,7 +3,7 @@
 // See LICENSE file in the project root for full license information.
 
 /********************************************************************
- * This code was auto-generated from key_vault.1 DL and shall not be
+ * This code was auto-generated from cipher.1 DL and shall not be
```

Comment:
>cipher.1 DL [](http://example.com/codeflow?start=37&length=11)

I think this has potential to be a tad confusing. cipher is a capability within the key_vault package but the name of this source file is only generated from the cipher model.

Reply:
You are right, I need to change the name of this file and remove all reference to key_vault from it. I'll rollback this change and fix it in another PR.